.github/workflows/skywalking.yaml

```diff
@@ -826,7 +826,7 @@ jobs:
           KUBERNETES_VERSION: ${{ matrix.versions.kubernetes }}
           ALS_ANALYZER: ${{ matrix.analyzer }}
         with:
-          e2e-file: $GITHUB_WORKSPACE/test/e2e-v2/cases/istio/als/e2e.yaml
+          e2e-file: test/e2e-v2/cases/istio/als/e2e.yaml
```

Comment:
What was the `$GITHUB_WORKSPACE/` for? @kezhenxu94

Reply:
> What was the `$GITHUB_WORKSPACE/` for? @kezhenxu94

Just to make sure the path is absolute file path, if the e2e-file is relative to the workspace folder it’s OK to remove this